docs/podman-create.1.md

```diff
@@ -268,26 +268,28 @@ The following example maps uids 0-2000 in the container to the uids 30000-31999
 
 Add additional groups to run as
 
-**--healthcheck-command**=*command*
+**--health-cmd**, **--healthcheck-command**=*command*
 
 Set or alter a healthcheck command for a container. The command is a command to be executed inside your
 container that determines your container health.  The command is required for other healthcheck options
 to be applied.  A value of `none` disables existing healthchecks.
 
-**--healthcheck-interval**=*interval*
+Healthchecks currently only work when running as root.
```

Comment:
Actually no longer the case - Giuseppe is fixing this with #3529

Reply:
Removed these.